mk/dependencies/jsonnet-bundler.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+set -ex
+
+OUTPUT_DIR=/home/codespace/.kuma-dev/kuma-release-none./bin
+VERSION="0.5.1"
+
+VERSION_NAME="jb-linux-amd64"
+jb="${OUTPUT_DIR}/jb"
+if [ -e "${jb}" ] && [ "$(${jb} --version)" == v${VERSION} ]; then
```

Comment:
I don't understand why `"$(${jb} --version)"` keeps being evaluated to empty string so it is always a false condition

Reply:
It was being output in the std err .
It's fixed